src/extends/room/meta.js

```diff
@@ -164,7 +164,13 @@ Room.isHallway = function (name) {
 }
 
 Room.isClaimable = function (name) {
-  if (!Game.map.isRoomAvailable(name)) {
+  if (Game.gcl.level >= GCL_NOVICE && Game.map.getRoomStatus(name).status === 'novice') {
```

Comment:
This condition will prevent Quorum from claiming any rooms in novice areas if its GCL is GCL_NOVICE or higher. This means that it can't claim a third room (or even a second one if it spawned in the novice area with GCL 3).